### PR TITLE
Changed action/about to dark text on light background

### PR DESCRIPTION
**Pull request by arianazhu**

### CODE REVIEW

**Review by fgoyal — APPROVED** [review]

File: src/main/webapp/styles/action.css [hunk 1]
```diff
@@ -18,4 +18,19 @@
...
     width: 20em;
+}
+
+.page-header {
+    /* margin: 0 10em 0 10em !important; */
+    background: linear-gradient(
+        to bottom, 
+        var(--dark-main-1),
+        10%,
+        var(--light-bg-1)
```
Comment (fgoyal) [inline src/main/webapp/styles/action.css:29]: are the two pages supposed to have different color gradients?

Comment (arianazhu) [reply src/main/webapp/styles/action.css:29]: yeah but I can change that if we want

**Review by maxhchen — APPROVED** [review]

File: src/main/webapp/styles/action.css (hunk 1 above)
Comment (fgoyal) [reply src/main/webapp/styles/action.css:29]: hmm we can ask Maxwell for his opinion too, but I think I like the Action page color more than the About Us page

Comment (maxhchen) [reply src/main/webapp/styles/action.css:29]: I like the About Us Page color more in general, but I think the Action Page color would fit better for our site

Comment (fgoyal) [reply src/main/webapp/styles/action.css:29]: Yup same, the About Us color's definitely prettier, but the green Action color fits better.

Comment (fgoyal) [reply src/main/webapp/styles/action.css:29]: I'm not entirely sure how I feel about the gradient though? We only have solid colors everywhere else, so it feels a bit weird. Definitely open for debate though

Comment (arianazhu) [reply src/main/webapp/styles/action.css:29]: oh no problem I'll change both of them to green! and yeah I wasn't sure about the gradient- I was trying to get it to look more uniform with the homepage but I can take that out too

Comment (arianazhu) [reply src/main/webapp/styles/action.css:29]: fixed!

**Pull request merged**